--- apply_report250	2019-12-19 15:51:01.929644805 +0000
+++ apply_report500	2019-12-19 15:52:00.926310067 +0000
@@ -1,4 +1,4 @@
-        host → 1.15.0       USB 72 URB_CONTROL out  26945508581da676 
+        host → 1.15.0       USB 72 URB_CONTROL out  26942508681da676 
         host → 1.15.0       USB 72 URB_CONTROL out  269015087cfda676 
       1.15.0 → host         USB 72 URB_CONTROL in   268f850884fdb6c6
         host → 1.15.0       USB 72 URB_CONTROL out  269045087c4da676 
